v2v-helper/vm/vmops.go

```diff
@@ -21,7 +27,7 @@ import (
 type VMOperations interface {
 	GetVMInfo(ostype string) (VMInfo, error)
 	GetVMObj() *object.VirtualMachine
-	UpdateDiskInfo(vminfo VMInfo) (VMInfo, error)
+	UpdateDiskInfo(vminfo VMInfo, namespace string) (VMInfo, error)
```

Comment:
Like we discussed before, we don't need the namespace as we are already inside the correct namespace. 
In-cluster credentials have namespace, so the client we build will have correct namespace.

Reply:
Sure